src/decrypt.rs

```diff
@@ -90,6 +91,52 @@ impl Decryptor {
             Err(anyhow!("decrypt failed!"))
         }
     }
+
+    pub async fn get_decrypt_key(
+        &self,
+        descriptor: &OciDescriptor,
+        decrypt_config: &str,
+    ) -> Result<Vec<u8>> {
+        if !self.is_encrypted() {
+            return Err(anyhow!("unencrypted media type: {}", self.media_type));
```

Comment:
nit: `Err(anyhow!(...))` can be replaced by `anyhow::bail!(...)`, which seems more concise. When `anyhow::bail` is included, here may be
```rust
bail!("unencrypted media type: {}", self.media_type)
```

Reply:
Yes, good suggestion, add one commit to replace to `bail` macro now.